### ISSUE TITLE
Complexes table not showing up on some entry pages

### ISSUE THREAD

**Issue by karenross** [PROconsortium/PRoteinOntology#180]
For some terms that are part of complexes (but not all), the table showing the complexes is not appearing on the entry page.
For example, on the page for PR:000025934 there is no complexes table, but on the page for PR:Q15796 there is a complexes table.

**Comment by nataled** [comment]
The org-gene term has the table, but not the actual component. Likely because complex components don't show the complexes indicated; that information is within the complex.

@nataled to find examples of complexes with components that are either org-seq, seq, or others.

**Comment by nataled** [comment]
The examples. Categories given in parentheses refer to the identifer(s) in the list (component categories are outside the parentheses). I provided two examples for each combination. If you see only one, that means there's literally just one case like that.

Simple and common cases, where all components are of the same type:

(complex) gene PR:000025784 PR:000025922
(complex) sequence PR:000027243 PR:000027245
(complex) modification PR:000025402 PR:000025493
(org-complex) org-gene PR:000024934 PR:000024937
(org-complex) org-sequence PR:000026130 PR:000026132
(org-complex) org-modification PR:000025765 PR:000025773

Simple uncommon cases:

(org-complex) org-complex PR:000050359 PR:000050360
(org-complex) org-family PR:000050361
(org-complex) org-seqgroup PR:000027030 PR:000027031

Mixed component types:

(complex) complex gene PR:000027079
(complex) complex gene modification PR:000025974
(complex) complex modification PR:000027080 PR:000027081
(complex) family gene modification PR:000026023
(complex) family modification PR:000029190
(complex) gene modification PR:000025980 PR:000025983
(complex) gene sequence PR:000026028 PR:000026045
(complex) modification sequence PR:000025967 PR:000025968
(org-complex) org-complex org-family PR:000027111
(org-complex) org-complex org-family org-gene org-modification PR:000036114 PR:000036117
(org-complex) org-complex org-gene PR:000028467 PR:000050165
(org-complex) org-complex org-gene org-modification PR:000025965 PR:000050167
(org-complex) org-complex org-modification PR:000028460 PR:000028461
(org-complex) org-family org-gene PR:000027102 PR:000035730
(org-complex) org-family org-gene org-modification PR:000028458 PR:000028465
(org-complex) org-family org-gene org-modification union PR:000026295
(org-complex) org-family org-gene org-seqgroup PR:000050161 PR:000050162
(org-complex) org-family org-seqgroup PR:000050159 PR:000050160
(org-complex) org-gene org-modification PR:000025716 PR:000025933
(org-complex) org-gene org-modification org-sequence PR:000026806 PR:000027174
(org-complex) org-gene org-sequence PR:000025037 PR:000025038
(org-complex) org-modification org-sequence PR:000025940 PR:000025944

**Comment by nataled** [comment]
@Julie-Cowart to review above list.

**Comment by Julie-Cowart** [comment]
I can see in the code that this was only ever implemented for genes and
therefore would only show up on the gene or organism-gene page. It lists
all children terms that are in a complex (and which one). However, I can
see that it would be useful so I will look into adding it to all pages.
Adding a list of complexes that the exact term is in is easy so I will
start there. Including those of children terms may be possible but may have
issues for some terms with non-gene parents, etc.

**Comment by Julie-Cowart** [comment]
Turned out easier than I thought once I got into it and I was able to copy
what was done for genes easily enough. Now not only will genes and
org-genes have this table but all terms should if they have a forms table
(family and some top level terms don't). And it will include the given term
and all its children (for example this sequence term and its org-seq
children https://proconsortium.org/app_test/entry/PR%3A000027157) in the
same way that the protein forms table does.  In fact, the green numbers
indicating complexes in the protein forms table should cause the
corresponding entries to highlight just like they do for the gene pages
when before that did nothing since the complexes were missing.
